### PR TITLE
no-test-jar=false

### PR DESCRIPTION
**Pull request by jglick**
Downstream of https://github.com/jenkinsci/plugin-pom/pull/38.

@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -72,20 +77,6 @@
     <build>
         <plugins>
             <plugin>
-                <artifactId>maven-release-plugin</artifactId>
-                <version>2.5</version>
```
Comment (jglick) [inline pom.xml]: Inherit from parent.

File: pom.xml [hunk 2]
```diff
@@ -99,13 +90,6 @@
                     <additionalparam>-Xdoclint:none</additionalparam>
                 </configuration>
             </plugin>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>findbugs-maven-plugin</artifactId>
-                <configuration>
-                    <skip>true</skip>
```
Comment (jglick) [inline pom.xml]: Better to suppress failures with the standard property.

**Comment by pkuczynski** [comment]
@jglick seems that the build fails due JavaDoc error. Would you mind having a look?

**Comment by jglick** [comment]
Seems to pass on JDK 8 but not 7; will adjust CI settings.

**Comment by abayer** [comment]
@jglick doesn't it need to pass with 7 to do the release?

**Comment by jglick** [comment]
> doesn't it need to pass with 7 to do the release?

Only if you

    JAVA_HOME=…/jdk7 mvn release:{prepare,perform}

At any rate I think JDK 8 is the default tool for doing builds.

**Comment by abayer** [comment]
Ah. I've had some problems with jdk8 targeting 7 but that may just be
Groovy weirdness. So :bee:

**Comment by jglick** [comment]
@reviewbybees done

**Comment by jglick** [comment]
Needs fix, do not merge.

**Comment by pkuczynski** [comment]
@jglick OK

**Comment by jglick** [comment]
Now should be OK.

**Comment by pkuczynski** [comment]
Merging then...

**Comment by pkuczynski** [comment]
Should I release new version too?

**Comment by jglick** [comment]
Well there is no user benefit, I just filed this so that you would not accidentally try to update the parent POM and not notice that you needed to switch the `test-jar` format.

**Pull request merged**